src/main/webapp/availability.html

```diff
@@ -39,10 +39,16 @@
       </nav>
       <br>
       <h1 class="text-center">Availability</h1>
-      <h5 class="text-center lead">Update your availability to be an Interviewer for this week below:</h5>
+      <h5 class="text-center lead">
+        Update your availability to be an Interviewer for the week below. Remember that an interview
+        lasts for an hour.
+      </h5>
+      <br>
+      <div id="page-buttons">
+        <button id="back-button" class="btn btn-primary mb-2 btn-sm" onclick="goBack()">&lt Previous Week</button>
+        <button id="forward-button" class="btn btn-primary mb-2 btn-sm" onclick="goForward()">Next Week &gt</button>
```

Comment:
Can we get these a different style from the "submit" buttons we have everywhere else?
https://getbootstrap.com/docs/4.0/components/buttons/
l'm a fan of 
<button type="button" class="btn btn-light">Light</button>
or 
one of the outline buttons.

Reply:
I thought the light was too light / hard to read so I went with a dark grey.